sycl/test/esimd/block_load_store.cpp

```diff
@@ -1,4 +1,4 @@
-// RUN: %clangxx -fsycl -fsyntax-only %s
+// RUN: not %clangxx -fsycl -fsyntax-only -Wno-unused-command-line-argument %s 2>&1 | FileCheck %s --implicit-check-not="warning:" --implicit-check-not="error:"
```

Comment:
Sorry I don't think I understand `--implicit-check-not="error:"` here and in the below tests. Aren't we causing an error in the newly added negative tests because of no matching function?

Reply:
Test gets some negative checks. implicit-check-not=error means that the test catches some errors on negative test-cases, but other errors if there are any would be considered as test fail.

Reply:
Ah got it, cool!